src/providers/node/mod.rs

```diff
@@ -95,7 +92,17 @@ impl Provider for NodeProvider {
         }
 
         // Install
-        let mut install = Phase::install(NodeProvider::get_install_command(app));
+        let corepack = NodeProvider::uses_corepack(app)?;
+        let mut install = Phase::install(if corepack {
+            Some("npm install -g corepack && corepack enable".to_string())
+        } else {
+            NodeProvider::get_install_command(app)
+        });
+
+        if corepack {
+            install.add_cmd(NodeProvider::get_install_command(app).unwrap());
+        }
```

Comment:
Last nit: Please remove the unwrap here and check if there is actually an install command

Reply:
Oh, didn't see this, will fix.